docs/SDK_v2.md

```diff
@@ -88,3 +91,32 @@ Event -> Message -> bits
 bits -> Message -> Event
 
 bit -> Message -> bits
+
+## Interfaces (Current pre-v2)
+
+```
+Client --> Transport (via TransportBinding) -> Protocol (implements Sender, Receiver)* -> Write<DataStructure>
+```
+
+`Write<DataStructure>` functions read a `binding.Message` and write what is
+found into the [3pl][3pl] data structure. For example, `nats.WriteMsg` writes
+the message into a `nats.Msg`, or `http.WriteRequest` writes message into a
```

Comment:
Now it's called `WriteHttpRequest`

Reply:
I am documenting what I want, not what is...